### DISCUSSION TITLE
Combining fonts.     Getting Korean into my existing  euro/japanese font

### DISCUSSION THREAD

**Discussion by Tonkabot1** [fontforge/fontforge#5129] [Q&A]
I want to add the Korean symbols to my font.    
I figured out how to cut and paste them in (many at a time, thank you!) , but the added symbols are smaller than the existing ones.
I can transform the new characters (scale them up to fit (175% !)), but when I scale them they lose some sort of width info, and groups of the new symbols are all mashed together like their width info is wrong  (not by 175%, by a much smaller but still unacceptable amount).
My font that I started with supports 13 languages including english, a bunch of europe, Arabic, Vietmanese, and Japanese.
But getting the Korean into it at about the same size with normal spacing is not happening.
I got it in with correct spacing, but it the chars are shrunken (to about %40 or so) from what they should be.

Is there a way to do this?

Oh, my fonts are all truetype.

**Comment by skef** [comment]
Are you using the UI for the transform, or scripting?

**Comment by Tonkabot1** [comment]
After I select the entire Korean bunch of symbols,   I do a right-click
transform.  So I guess I used the UI.
I looked at the scripting briefly, but i'd have to read up a lot on it to
attempt anything.

Thanks,  Brynn

**Comment by skef** [comment]
The UI might be best for this.
Since you're cutting pasting are you sure its the transform that's introducing the problem? If you're comparing to the "source" Korean font could there just be info (like kerning classes) that isn't getting moved over?

**Comment by ctrlcctrlv** [comment]
Since Korean is a uniform width script, why is it not acceptable to use Set Right Bearing after the transform? Also you are checking Transform Widths, right?

**Comment by Tonkabot1** [comment]
How do I check the widths, and how do I set right bearing, and what does it
do?

How would a simple script look that for all the range from \uac00 to \uafff
transform 166% bigger and fix whatever makes the spacing bad ?

**Reply by ctrlcctrlv** [reply]
Ctrl+R brings up «Set RBearing…» dialog.

**Comment by Tonkabot1** [comment]
That worked.    I scaled the (that part of the) font up 166%, then also the
right bearing up by 166%, and the result is what I am looking for!
Thanks.
